src/index.js

```diff
@@ -0,0 +1,27 @@
+import './style.css';
+
+const dataScore = [
+  { name: 'name', score: 100 },
+  { name: 'name', score: 53 },
+  { name: 'name', score: 67 },
+  { name: 'name', score: 51 },
+  { name: 'name', score: 23 },
+  { name: 'name', score: 98 },
+];
+
+function tableLoad() {
+  const table = document.querySelector('.table-score-board');
+
+  dataScore.forEach((data) => {
+    const tr = document.createElement('tr');
+    tr.className = 'row-table';
+    const td = document.createElement('td');
+    td.className = 'data-table';
+    td.innerHTML = `${data.name}: ${data.score}`;
+
+    tr.appendChild(td);
+    table.appendChild(tr);
+  });
+}
```

Comment:
- [ ] great job on implementing JS in your project 👍 however the goal of the assignment is that we get familiar with ES6 modules to write modular JavaScript with proper ES6 syntax. part of that means using arrow functions instead of the function keyword, which will look something like below:
```
const tableLoad = () => {
  your code....
}

export default tableLoad();
```

Reply:
Thanks sir, for your feedback! I have made the changes!